<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QMainWindow" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1221</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OMA-Main</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(117, 127, 110);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="go_to_raiz">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>301</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>RAIZ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="operaciones">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>301</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>OPERACIONES BASICAS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="porcentaje">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>301</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>PORCENTAJE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="potencia">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>301</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>POTENCIA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="estadistica">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>301</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>ESTADISTICA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="salir">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>210</y>
      <width>301</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>SALIR</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1221</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <widget class="QMenu" name="menuContact_US">
     <property name="title">
      <string>Contact US</string>
     </property>
     <addaction name="actionRogelio15021515_gmail_com"/>
    </widget>
    <addaction name="menuContact_US"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRogelio15021515_gmail_com">
   <property name="text">
    <string>rogelio15021515@gmail.com</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
